test/es6/rlexe.xml

```diff
@@ -1070,7 +1070,7 @@
   <test>
     <default>
       <files>bug_OS_2553885.js</files>
-      <tags>exclude_win7,BugFix</tags>
+      <tags>exclude_win7,BugFix,exclude_xplat</tags>
```

Comment:
Copy over the comment:

es6/unicode_6_identifiers.js

Looked off line, there is a Windows/Linux diff in  bool IsIdStart(codepoint_t codepoint)  implementation. @digitalinfinity will investigate.